src/commands/src/scripts/local-build/utils/get-args.ts

```diff
@@ -2,7 +2,15 @@ export function getArgs() {
   const helpIdx = process.argv.findIndex((str) => str === '--help' || str === '-h');
 
   if (helpIdx !== -1) {
-    console.log('TODO: Fill this out');
+    console.log(`
+      This command builds and runs the Autobuilder for local debugging.
+
+      Flags:
+      --repo-owner,  -o  (required)       The owner of the repo. Typically this is 'mongodb' or '10gen'. This should be your username for a fork.
+      --repo-name,   -n  (required)       The name of the repo e.g. docs-java or cloud-docs.
+      --directory,   -d  (optional)       The project directory path for a monorepo project.
```

Comment:
Could you specify that this does not include a leading forward slash? 
ie the directory arg should be `cloud-docs` not `/cloud-docs`

Reply:
Hmm, we could do this! It would fail right now if it had a leading slash because it is used throughout the pipeline in different ways, sometimes adding the slash, sometimes not.